### DISCUSSION TITLE
Demo for godot 3.5 windows

### DISCUSSION THREAD

**Discussion by Ehsanwww** [j20001970/GDMP-demo#14] [General]
Hello @j20001970  thanks for great asset, as I see in repository 3.5, there is no build for windows, can you please provide one ? I also use  build GDMP for godot 3.5 windows but got error for windows. thanks again.

**Comment by j20001970** [comment]
Sorry for much belated reply, since GDMP v0.4, the development focus has been moved to Godot 4.x (GDExtension), thus original GDNative version of demo project is no longer provided. We encourage users migrating to Godot 4.x if possible, prebuilt library for Windows is also available in demo project.

You will need to build from source on GDMP `3.x` branch if you decide to stay on Godot 3.x. You can still report issues for GDNative version if you run into problems while building or using GDMP.

Apologize again for the inconvenience, hope this can answer your questions.
